src/de/calcom/cclib/text/FindReplaceDialog.java

```diff
@@ -618,7 +618,7 @@ private int findNext() {
     	if (searchTerm.isEmpty())
     		return -1;
     	
-        int start = -1; // -1 means not found.
```

Comment:
If you want to be perfect, replace 1 by constant value final static private inr NOT_FOUND=-1 to eliminate the comment